README.md

```diff
@@ -404,7 +404,8 @@ Sends a Telegram message.
 {
     "telegram": {
         "token": "telegram bot token",
-        "bot_ident": "token used to activate bot in a group"
+        "bot_ident": "token you choose to activate bot in a group"
+        "chatfile": "path to file where chat id`s are saved"
```

Comment:
Can you make it clear that this is optional and change `id<backtick>s` => `ids`?